### ISSUE TITLE
Fix line breaking

### ISSUE THREAD

**Issue by koppor** [JabRef/jabref#114]
Somehow, the abstract and keyword fields are incorrectly wrapped. See https://sourceforge.net/p/jabref/discussion/318825/thread/9609d582/ for details.

**Comment by stefan-kolb** [comment]
The bug should be caused by `FieldContentParser`

```text
abstract = {The Quick Brown Fox Jumps Over The Lazy Dog. The Quick Brown Fox Jumps\r\n
	\tOver The Lazy Dog. The Quick Brown Fox Jumps Over The Lazy Dog. The
```

```java
 if (content.length() > i + 1 && content.charAt(i + 1) == '\t' && (content.length() == i + 2 || !Character.isWhitespace(content.charAt(i + 2)))) {
// We have either \n\t followed by non-whitespace, or \n\t at the
// end. Both cases indicate a wrap made by JabRef. Remove and insert space if necessary.
// @formatter:on

content.deleteCharAt(i); // \n
content.deleteCharAt(i); // \t
```

**Comment by stefan-kolb** [comment]
The question here is, which fields should accept line breaks?

**Comment by koppor** [comment]
All multiline fields :joy:

Arent't they enlisted in the code somewhere?

Multiline: Abstract, Review
Unsure: comment
Non-multiline: all others

Side note: No one has complained about the difference between UI and serialization. To have the `.bib` files editable, I would keep the autowrap-feature (refs #123).

**Comment by sbitzer** [comment]
Hi, I'm very much for keeping some line breaks in multiline fields. I previously used the review field to save my own thoughts about a paper. For some entries this has been quite a bit of text which I had separated into paragraphs by putting an empty line between paragraphs in the entry editor. I recently noticed that linebreaks are removed when the database is saved. When it is saved the first time, my empty lines are gone. When it is saved the second time, the remaining linebreaks are gone, too. Needless to say that I found that all my old reviews are only a single paragraph mess now. I don't think that this was an issue earlier and I'm a bit upset about the change in behaviour.

My jabref version is 2.10b2.

**Comment by koppor** [comment]
Related issue: https://sourceforge.net/p/jabref/feature-requests/854/

**Comment by stefan-kolb** [comment]
The writer should also not write line breaks for non-multiline fields...

**Comment by lenhard** [comment]
Fixed #187
